### ISSUE TITLE
Pod URL is getting redirected to Home page (included Readme.md in my zipfile)

### ISSUE THREAD

**Issue by cai22r** [CocoaPods/trunk.cocoapods.org#262]
Report
What did you do?

pod trunk push .podspec

What did you expect to happen?

MicrosoftMapsSDK should be displayed in cocoapods.
The https://cocoapods.org/pods/microsoftMapsSDK should redirect to cocoapods and the LandingPage displayed based on the included readme.md file

What happened instead?

Pod published Successfully, but URL is getting redirected to Home page URL.
Further the artifact MicrosoftMapsSDK is not displayed in cocoapods.

CocoaPods Environment
3.0.2

**Comment by orta** [comment]
If it's a closed source pod, then it could likely be https://github.com/CocoaPods/cocoapods-metadata-service/issues/1 or https://github.com/CocoaPods/cocoapods-metadata-service/issues/2

**Comment by cai22r** [comment]
> If it's a closed source pod, then it could likely be [CocoaPods/cocoapods-metadata-service#1](https://github.com/CocoaPods/cocoapods-metadata-service/issues/1) or [CocoaPods/cocoapods-metadata-service#2](https://github.com/CocoaPods/cocoapods-metadata-service/issues/2)

For the search indexing part, is it related as well, or i just have to wait for a period of time?

**Comment by orta** [comment]
Nope, indexing for the website is separate, [looks like that's daily](https://github.com/CocoaPods/indexer.cocoapods.org/blob/master/src/index.ts#L33) - I wonder if that could move to hourly
